jhove-modules/src/main/java/edu/harvard/hul/ois/jhove/module/tiff/GPSInfoIFD.java

```diff
@@ -15,6 +15,7 @@
 public class GPSInfoIFD
     extends IFD
 {
+	private static final String period = ".";
```

Comment:
Unnecessary constant. It's less code and clearer to read without this, and doesn't appear to provide any maintenance benefits since the value isn't going to change. And if it were to, the name of the variable would ideally be something more along the lines of `GPS_VER_SEPARATOR` so that the value could change without also needing the name to.